tests/Fixer/FunctionNotation/NullableTypeDeclarationForDefaultNullValueFixerTest.php

```diff
@@ -500,4 +500,27 @@ function foo2(?string &/*comment*/$param2 = null) {}
         yield [$cases[0], $cases[1]];
         yield [$cases[1], $cases[0], ['use_nullable_type_declaration' => false]];
     }
+
+    /**
+     * @dataProvider provideFix82Cases
+     * @requires PHP 8.1
+     */
+    public function testFix81(string $expected): void
+    {
+        $this->doTest($expected);
+    }
+
+    public function provideFix82Cases(): iterable
```

Comment:
```suggestion
    public function provideFix81Cases(): iterable
```

Reply:
:+1:  thanks, done, also update the PHPDoc for the provider